### PR TITLE
Use toggle buttons in Cartographic Limit Rectangle sandcastle example

### PR DESCRIPTION
**Pull request by hpinkos**
`addToggleButton` is preferred for this kind of toggling behavior because it shows the button state, where `addToolbarButton` does not

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @hpinkos!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/bounds-rectangle-example/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by hpinkos** [comment]
@ggetz can you take a quick look and merge this?

File: Apps/Sandcastle/gallery/Cartographic Limit Rectangle.html [hunk 1]
```diff
@@ -55,25 +55,21 @@
     }));
 }
 
-Sandcastle.addToolbarButton('Show/Hide Bounds', function() {
-    var rectanglesLength = rectangles.length;
-    for (var i = 0; i < rectanglesLength; i++) {
-        var rectangleEntity = rectangles[i];
-        rectangleEntity.show = !rectangleEntity.show;
+Sandcastle.addToggleButton('Enabled', true, function(checked) {
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/Cartographic Limit Rectangle.html]: Can we say "Limiter enabled" just to clarify? Otherwise looks good. :+1:

**Comment by hpinkos** [comment]
@ggetz ready

**Comment by ggetz** [comment]
Thanks @hpinkos!

**Pull request merged**